### DISCUSSION TITLE
fetchOperation status undefined

### DISCUSSION THREAD

**Discussion by enri90** [urql-graphql/urql#3140] [Q&A] [answered]
Hi, when connection the graphql server is not reachable it gives me this error message blocking the application
how can i intercept this error message and handle the situation with an alert or dedicated page in vitejs ssr

```
TypeError: Cannot read properties of undefined (reading 'status')
    at fetchOperation (/test-vite/urql/node_modules/@urql/core/src/internal/fetchSource.ts:148:1)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at /test-vite/urql/node_modules/wonka/dist/wonka.js:342:20
```

**Comment by kitten** [comment, marked as answer]
If I'd have to guess, you're likely on a version before [`@urql/core@4.0.3`](https://github.com/urql-graphql/urql/releases/tag/%40urql%2Fcore%404.0.3), so you should definitely upgrade, since that issue has been fixed in that release (and an additional one in `4.0.4` btw)

Also, just another reminder here for everyone that bindings (and exchanges) depend on `@urql/core`, so you _may not_ have to install it manually and add it to your own dependencies. Although there is no harm in it. All package managers support updating transitive dependencies, e.g. in `pnpm` that's `pnpm update @urql/core`.

The only note as per the docs and the [release announcement](https://github.com/urql-graphql/urql/issues/3114), you'll sometimes want to deduplicate dependencies (instructions in the release announcement), so you don't have two installations of `@urql/core`.

**Reply by enri90** [reply]
thanks it works
